doc/man3/BIO_f_prefix.pod

```diff
@@ -0,0 +1,70 @@
+=pod
+
+=head1 NAME
+
+BIO_f_prefix, BIO_set_prefix, BIO_set_indent, BIO_get_indent
+- prefix BIO filter
+
+=head1 SYNOPSIS
+
+ #include <openssl/bio.h>
+
+ const BIO_METHOD *BIO_f_prefix(void);
+ long BIO_set_prefix(BIO *b, const char *prefix);
+ long BIO_set_indent(BIO *b, long indent);
```

Comment:
Emilia had a thing about not using long for new code because it doesn't buy you anything, i.e. we can't guarantee its any longer than int

Reply:
These are macros wrappers around `BIO_ctrl()`, which is declared like this:

https://github.com/openssl/openssl/blob/3c86a2b5c81fd1d39155e4fea35a55c29f9bc4c4/include/openssl/bio.h#L594